README.md

```diff
@@ -5,15 +5,19 @@
 [![Build Status](https://travis-ci.org/blox/blox.svg?branch=master)](https://travis-ci.org/blox/blox)
 
 Blox provides open source schedulers optimized for running applications on Amazon ECS. Developers now have greater control over how their applications are deployed across clusters of resources, run and scale in production, and can take advantage of powerful placement capabilities of Amazon ECS.
+
 Blox is being delivered as a managed service via the Amazon ECS Console, API and CLIs. Blox v1.0 provides daemon scheduling for Amazon ECS. We will continue to add additional schedulers as part of this project.
-Blox schedulers are built using AWS primitives, and the Blox designs and code are open source. If you are interested in learning more or collaborating on the designs, please read the [design](docs/daemon_design.md) 
-If you are currently using Blox v0.3, please read the [FAQ](docs/faq.md) 
+
+Blox schedulers are built using AWS primitives, and the Blox designs and code are open source. If you are interested in learning more or collaborating on the designs, please read the [design](docs/daemon_design.md)
+
+If you are currently using Blox v0.3, please read the [v0.3 FAQ](docs/faq.md)
 
 ### Contact
 
 * [Gitter](https://gitter.im/blox)
 * [Planning/Roadmap](https://github.com/blox/blox/milestones)
 * [Issues](https://github.com/blox/blox/issues)
+* [Blox Homepage](https://blox.github.io)
```

Comment:
Recommend adding this link to the list once the homepage is updated, so that we do not confuse new readers.